src/main/java/org/veupathdb/lib/gradle/container/tasks/base/exec/ExecAction.kt

```diff
@@ -137,16 +137,16 @@ abstract class ExecAction : Action() {
 
       // Configure the pipe for the command's stdout.  If no custom
       // configuration is provided, pipe stdout to /dev/null
-      var pipe = getStdOutRedirect() ?: RedirectConfig.toDevNull()
-      pipe.forStdOut(proc)
+      var pipe: RedirectConfig? = getStdOutRedirect() ?: RedirectConfig.toDevNull()
```

Comment:
I intentionally want `pipe` to be nullable here as I'm using it in a nullable context later in the file.